go.mod

```diff
@@ -3,7 +3,7 @@ module golang-online-payments
 go 1.14
 
 require (
-	github.com/adyen/adyen-go-api-library/v5 v5.1.0
+	github.com/adyen/adyen-go-api-library/v6 v6.0.0-20220705095509-7ba1b8ecdfc8 // indirect
```

Comment:
What does // indirect mean?

Reply:
@Kwok-he-Chu I have imported SDK v6 with `go get` instead of adding it into the `go.mod` file, the reason is Go SDK v6 is not released yet so I have used a specific Git commit to move forward. I plan to change it (adding the dependency into `go.mod` - direct) as soon as v6 is out